conda_build/config.py

```diff
@@ -822,9 +823,10 @@ def clean_pkgs(self):
 
     def copy(self):
         new = copy.copy(self)
-        new.variant = copy.deepcopy(self.variant)
+        # Use picke.loads(pickle.dumps(...) as a faster copy.deepcopy alternative.
+        new.variant = pickle.loads(pickle.dumps(self.variant, -1))
         if hasattr(self, "variants"):
-            new.variants = copy.deepcopy(self.variants)
+            new.variants = pickle.loads(pickle.dumps(self.variants, -1))
```

Comment:
Do you want a try except here that reverts back to deepcopy n case something cannot be pickled?

Reply:
Thanks for taking a look!
I don't think we should get anything in there that `pickle` won't be able to handle; but a try/except also wouldn't hurt, of course.